admin-print-action/shopify.extension.toml.liquid

```diff
@@ -1,12 +1,13 @@
 api_version = "2024-04"
 
 [[extensions]]
+# Name is not used in admin print actions, but cannot be removed because of validation requirements.
```

Comment:
Do we care to even mention this?

Reply:
I thought it might be useful if someone was trying to track down where it's used. I'm happy to remove it too.